API/NoDocs/Microsoft.AspNetCore.Testing/ServiceFakesExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Assembly 'Microsoft.AspNetCore.Testing'
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.AspNetCore.Testing;
```

Comment:
These extensions should be in the same namespace as their extended types, IWebHostBuilder and IHost (even if it means splitting the class). I'll take care of that.